### PR TITLE
Fix typos again

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -H`

Chore that needs to be done:

* [x] Add newsfragment `pipenv run towncrier create [issue_number].[type].rst`

Types are defined in the pyproject.toml, issue_numer either from issue tracker or the Pull request number

### CODE REVIEW

**Comment by fizyk** [comment]
newsfragment is missing

**Comment by fizyk** [comment]
Same as pr number :)

**Comment by kianmeng** [comment]
@fizyk Done.

File: pytest_postgresql/config.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def get_postgresql_option(option: str) -> Any:
 
 
 def detect_paths(load_paths: List[str]) -> List[Union[Path, str]]:
-    """Covnerts path to sql files to Path instances."""
+    """Converts path to sql files to Path instances."""
```
[the remark this answers is not in the document]
Comment (kianmeng) [reply pytest_postgresql/config.py]: Done.

```
$ pipenv run ruff check .
warning: The top-level linter settings are deprecated in favour of their counterparts in the `lint` section. Please update the following options in `pyproject.toml`:
  - 'select' -> 'lint.select'
warning: `one-blank-line-before-class` (D203) and `no-blank-line-before-class` (D211) are incompatible. Ignoring `one-blank-line-before-class`.
warning: `multi-line-summary-first-line` (D212) and `multi-line-summary-second-line` (D213) are incompatible. Ignoring `multi-line-summary-second-line`.
```

**Comment by fizyk** [comment]
Thank you @kianmeng

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**